### PR TITLE
Test db

### PR DESCRIPTION
**Pull request by PaulFarault**
#### Which issue(s) this PR fixes

Fixes None

#### Additional comments

- Refactor tests to enable query testing.
- Add tests for the `get_latest_success_component_version_log` query.

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: tdp/core/models/test_models.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        state="Success",
+        logs=b"operation log",
+    )
+
+    deployment_log.component_version.append(component_version_log)
+    deployment_log.operations.append(operation_log)
+
+    logger.info(deployment_log)
+    logger.info(operation_log)
+    logger.info(operation_log)
```
Comment (sergkudinov) [inline tdp/core/models/test_models.py:52]: You may mean `logger.info(component_version_log)` the second time?

**Review by sergkudinov — CHANGES_REQUESTED** [review]

File: tdp/core/models/test_models.py (hunk 1 above)
Comment (PaulFarault) [reply tdp/core/models/test_models.py]: Didn't spot it as it was already there but you're right. I fixed it!

**Review by sergkudinov — APPROVED** [review]

File: tdp/conftest.py [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
     DAG_DIRECTORY_NAME,
     DEFAULT_VARS_DIRECTORY_NAME,
     PLAYBOOKS_DIRECTORY_NAME,
     SCHEMA_VARS_DIRECTORY_NAME,
 )
+from tdp.core.models import Base
+
+
+sqlalchemy_logger = logging.getLogger("sqlalchemy.engine")
+sqlalchemy_logger.setLevel(logging.INFO)
```
Comment (rpignolet) [inline tdp/conftest.py:23]: Where this logger is used ?

Comment (PaulFarault) [reply tdp/conftest.py:23]: It is not used in my code, it is a way to define log level as "INFO" for sqlalchemy in those tests.

We could replace it with:

```py
logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
```

Comment (rpignolet) [reply tdp/conftest.py:23]: ok so no need to put it in a variable.

Comment (PaulFarault) [reply tdp/conftest.py:23]: Beyond the syntax, idk if we should keep it.

Comment (PaulFarault) [reply tdp/conftest.py]: I removed it.

**Review by rpignolet — APPROVED** [review]

**Pull request merged**